errors.go

```diff
@@ -0,0 +1,19 @@
+package srvclient
+
+import "fmt"
+
+type notFoundErr struct {
+	hostname string
+}
+
+// Error implements the error interface
+func (err *notFoundErr) Error() string {
+	return fmt.Sprintf("No SRV records for %q", err.hostname)
+}
+
+// IsNotFound returns true of the supplied error indicates no SRV records were
+// found
+func IsNotFound(err error) bool {
```

Comment:
I can do that but I just had to deal with that for the acme stuff and its annoying to type-cast to this, especially since there's nothing really useful on the struct.